### PR TITLE
remove figer print sign and ligitation will now show actuall material

### PR DESCRIPTION
**Pull request by UmairMukhtar**
Now showing real material fixed

### CODE REVIEW

**Comment by AltiMario** [comment]
if you align the api I can test online

File: app/api/src/services/litigation.service.ts [hunk 1]
```diff
@@ -400,6 +401,8 @@ export const getLitigationByCriteria = async (
           options && options.participant_id ? options.participant_id : false,
         ].filter(Boolean)
       );
+      const foundMaterial = await getMaterialById(result.rows[0].material_id);
+      result.rows[0]['material'] = foundMaterial;
       return result.rows[0];
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/api/src/services/litigation.service.ts]: done

**Comment by UmairMukhtar** [comment]
[search creations.webm](https://user-images.githubusercontent.com/34018917/210106945-5b22ca73-e9e6-472c-8ace-9c8c5911d118.webm)

File: app/web-frontend/src/pages/Creations/Home/index.jsx [hunk 2]
```diff
@@ -135,7 +155,19 @@ function Creations() {
...
+            variant="dark"
+            placeholder="Search Creations!"
+            name="creation"
+            onInput={(event_) => handleCreationInputChange(event_)}
+            autoComplete
+            onChange={onCreationSelect}
+            autoCompleteOptions={
+              getSuggestions(creationSuggestions)
+            }
+          />
```
[the remark this answers is not in the document]
Comment (UmairMukhtar) [reply app/web-frontend/src/pages/Creations/Home/index.jsx]: @huzaifa-99 now made sure that search box only appears when user is not logged in

**Comment by AltiMario** [comment]
All good to me, thanks for the nice video explanation

**Review by AltiMario — APPROVED** [review]

**Comment by UmairMukhtar** [comment]
Issues fixed
https://github.com/eLearningDAO/POCRE/issues/230
https://github.com/eLearningDAO/POCRE/issues/230

**Pull request merged**